### DISCUSSION TITLE
Upload field => hasMany

### DISCUSSION THREAD

**Discussion by wikoci** [payloadcms/payload#803] [Q&A] [answered]
its possible to link upload field type to many media ?
such as :

{
name:'posters',
type:'upload',
realtionTo:'media',
hasMany:true
}

**Comment by denolfe** [comment, marked as answer]
I think this could be better achieved by using an array with the upload field on each row.

**Reply by Ivan-Shalnov** [reply]
Using hasMany would be more convenient. Currently, accomplishing this with an array is inconvenient and time-consuming. Additionally, using an array creates an inconvenient API structure.

**Reply by jnovak-SM2Dev** [reply]
This definitely needs to be added. Using an array is a terrible interface. The user would need to add every image one at a a time in each block. They should be able to drag and drop lets say 10 images all at once and have them upload. It would be nice if there was an option to allow reordering for them as well.

**Comment by wikoci** [comment]
noted.
Also I can't upload many files to media collection in one request.

**Comment by jnovak-SM2Dev** [comment]
This definitely needs to be added. Using an array is a terrible interface. The user would need to add every image one at a a time in each block. They should be able to drag and drop lets say 10 images all at once and have them upload. It would be nice if there was an option to allow reordering for them as well.
